### ISSUE TITLE
Rule proposal: forced disabling reason

### ISSUE THREAD

**Issue by dartess** [eslint/eslint#13338]
**Please describe what the rule should do:**

oblige the developer to explain the reason for disconnecting the linter

**What category of rule is this? (place an "X" next to just one item)**

[ ] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Enforces code style (layout)
[x] Other (please specify:) —  reduce the number of thoughtless shutdowns of the linter; Improve code reading speed by other developers.

**Provide 2-3 code examples that this rule will warn about:**

```js
// fail
// eslint-disable-next-line no-new
new Swiper(slider);

// ok
// eslint-disable-next-line no-new -- saving the instance is not necessary, since subsequent control of the slider is not needed
new Swiper(slider);
```

**Why should this rule be included in ESLint (instead of a plugin)?**

I'm not sure, but it seems to me that this will reduce the number of incorrect, incomprehensible and lazy shutdowns of the linter

**Are you willing to submit a pull request to implement this rule?**

I doubt my competence, this rule does not look banal.

**Comment by kaicataldo** [comment]
This has already been implemented in 7.0.0. Please see [the migration guide](https://eslint.org/docs/user-guide/migrating-to-7.0.0#description-in-directive-comments) for more details.

**Comment by dartess** [comment]
@kaicataldo I probably did not explain well that I was proposing **a new rule**, not the ability to post comments.
Implemented the ability to write comments.
My suggestion is to add a rule that makes writing comments mandatory.

**Comment by kaicataldo** [comment]
I don't think this in scope for core rules and that a 3rd party plugin like [eslint-plugin-eslint-comments](https://github.com/mysticatea/eslint-plugin-eslint-comments) would be a a better fit for enforcing this.

**Comment by dartess** [comment]
from eslint-plugin-eslint-comments's repo:

>I think your proposed rule is already implemented in the master.
> However, it has not been released yet.

@kaicataldo thanks for the right direction.
